spec/cancan/ability_spec.rb

```diff
@@ -500,6 +501,19 @@ class Container < Hash
       expect(@ability.unauthorized_message(:update, :missing)).to be_nil
     end
 
+    it "uses model's name in i18n" do
+      class Account
+        include ActiveModel::Model
+      end
+
+      I18n.default_locale = :ja
+      I18n.backend.store_translations :ja,
+                                      activemodel: { models: { account: 'アカウント' } },
```

Comment:
could you please use english? You can specify it as `ja` but please use english words anyway. 
Maybe also add translations in another language `en` to see that they actually differ in your result.
```
I18n.backend.store_translations :de, activemodel: { models: { account: 'english name' } }
I18n.backend.store_translations :ja, activemodel: { models: { account: 'japanese name' } }
expect(...).to eq('japanese name...')
```